kolibri/plugins/learn/assets/src/vue/learn-page/allcontent.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+
+  <card-grid :header="'All Content'" v-if="all.content.length">
+
+    <div slot="headerbox" class="allnav">
+
```

Comment:
for (1), do you think this would suffice?

```html
<div id="all-content-grid">
  <div class="header">
    <h3>All Content</h3>
    <div slot="headerbox" class="allnav" role="navigation" aria-labelledby="all-content-grid">
    ...
```

For technical reasons (due to the slot mechanism), it's a harder to put the id on the heading tag itself.